composer.json

```diff
@@ -1,49 +1,116 @@
 {
-	"name": "3brs/sylius-payment-restrictions-plugin",
-	"type": "sylius-plugin",
-	"description": "Payment restriction plugin for Sylius",
-	"license": "MIT",
-	"require": {
-		"php": "^7.3|^8.0",
-		"sylius/sylius": "1.7.*|1.8.*|1.9.*|1.10.*"
-	},
-	"require-dev": {
-		"behat/behat": "^3.6.1",
-		"behat/mink-selenium2-driver": "^1.4",
-		"dmore/behat-chrome-extension": "^1.3",
-		"dmore/chrome-mink-driver": "^2.7",
-		"friends-of-behat/mink": "^1.8",
-		"friends-of-behat/mink-browserkit-driver": "^1.4",
-		"friends-of-behat/mink-extension": "^2.4",
-		"friends-of-behat/page-object-extension": "^0.3",
-		"friends-of-behat/suite-settings-extension": "^1.0",
-		"friends-of-behat/symfony-extension": "^2.1",
-		"friends-of-behat/variadic-extension": "^1.3",
-		"friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev",
-		"lakion/mink-debug-extension": "^2.0.0",
-		"phpspec/phpspec": "^7.0",
-		"phpstan/phpstan": "0.12.88",
-		"phpstan/phpstan-symfony": "^0.12.7",
-		"phpstan/phpstan-doctrine": "0.12.37",
-		"phpunit/phpunit": "^8.5",
-		"sensiolabs/security-checker": "^6.0",
-		"sylius-labs/coding-standard": "^4.0.0",
-		"symfony/browser-kit": "^4.4|^5.2",
-		"symfony/debug-bundle": "^4.4|^5.2",
-		"symfony/dotenv": "^4.4|^5.2",
-		"symfony/intl": "^4.4|^5.2",
-		"symfony/web-profiler-bundle": "^4.4|^5.2"
-	},
-	"prefer-stable": true,
-	"autoload": {
-		"psr-4": {
-			"ThreeBRS\\SyliusPaymentRestrictionPlugin\\": "src/",
-			"Tests\\ThreeBRS\\SyliusPaymentRestrictionPlugin\\": "tests/"
-		}
-	},
-	"autoload-dev": {
-		"psr-4": {
-			"App\\": "tests/Application/src/"
-		}
-	}
+  "name": "3brs/sylius-payment-restrictions-plugin",
+  "type": "sylius-plugin",
+  "description": "Payment restriction plugin for Sylius",
+  "keywords": [
+    "sylius",
+    "sylius-plugin",
+    "plugin",
+    "payment",
+    "restriction"
+  ],
+  "license": "MIT",
+  "require": {
+    "php": "^8.0",
+    "sylius/mailer-bundle": "^1.8 || ^2.0@beta",
+    "sylius/sylius": "^1.12",
+    "symfony/webpack-encore-bundle": "^1.15"
```

Comment:
to tady asi taky nepotřebujeme? oba bych přendal do dev

Reply:
Oboje jsem odebral, pustil všechno od začátku... a prošlo to. Takže to nepotřebujeme.

Nevím proč to má Sylius skeleton plugin v základu :thinking: https://github.com/Sylius/PluginSkeleton/blob/1.12/composer.json